2020F-usfca-btec-640.github.io/Gemfile

```diff
@@ -0,0 +1,30 @@
+source "https://rubygems.org"
```

Comment:
This file should not be a part of the PR. Please remove it so we don't have to deal with merge conflicts.